tools/linter/adapters/no_merge_conflict_csv_linter.py

```diff
@@ -0,0 +1,132 @@
+import argparse
+import concurrent.futures
+import json
+import logging
+import os
+import sys
+from enum import Enum
+from pathlib import Path
+from typing import Any, List, NamedTuple, Optional
+
+from ufmt.core import ufmt_string
+from ufmt.util import make_black_config
+from usort import Config as UsortConfig
+
+
+IS_WINDOWS: bool = os.name == "nt"
+
+
+def eprint(*args: Any, **kwargs: Any) -> None:
+    print(*args, file=sys.stderr, flush=True, **kwargs)
+
+
+class LintSeverity(str, Enum):
+    ERROR = "error"
+    WARNING = "warning"
+    ADVICE = "advice"
+    DISABLED = "disabled"
+
+
+class LintMessage(NamedTuple):
+    path: Optional[str]
+    line: Optional[int]
+    char: Optional[int]
+    code: str
+    severity: LintSeverity
+    name: str
+    original: Optional[str]
+    replacement: Optional[str]
+    description: Optional[str]
+
+
+def as_posix(name: str) -> str:
+    return name.replace("\\", "/") if IS_WINDOWS else name
+
+
+def format_error_message(filename: str, err: Exception) -> LintMessage:
+    return LintMessage(
+        path=filename,
+        line=None,
+        char=None,
+        code="MERGE_CONFLICTLESS_CSV",
+        severity=LintSeverity.ADVICE,
+        name="command-failed",
+        original=None,
+        replacement=None,
+        description=(f"Failed due to {err.__class__.__name__}:\n{err}"),
+    )
+
+
+def check_file(filename: str) -> List[LintMessage]:
+
+    with open(filename, "rb") as f:
+        original = f.read().decode("utf-8")
+    replacement = ""
+    with open(filename, 'r') as f:
+        lines = f.readlines()
+        for line in lines:
```

Comment:
Hmm, is this going to add 3 new lines to the file every time I run lintrunner, so the file would keep growing.  I need to try this out locally :)

Reply:
It does not do this, at least when I tested this. Strip should return an empty line for the new lines